etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
(Just a reminder that you'll have to run it against production DB, running it against local DB or staging would produce different variable ids).